pytypes/type_util.py

```diff
@@ -21,7 +21,7 @@
 import typing
 import collections
 import weakref
-from inspect import isfunction, ismethod, isclass, ismodule, stack
+from inspect import currentframe, isfunction, ismethod, isclass, ismodule, stack
```

Comment:
I think `stack` can be removed here then.

Reply:
Done.